### DISCUSSION TITLE
I cannot load the dependencies

### DISCUSSION THREAD

**Discussion by Dominic-Kaemereit** [Minestom/Minestom#1715] [Help] [answered]
Hello,
I can not load the dependencies unfortunately.

```
repositories {
    maven {
        name = "jitpack"
        url = uri("https://jitpack.io")
    }
}

dependencies {
    implementation ("com.github.Minestom.Minestom:Minestom:-SNAPSHOT")
}
```

![Bild_2023-01-25_172509231](https://user-images.githubusercontent.com/65771707/214618953-07e1389c-22a3-4b00-9db7-a074d6dd6b0a.png)
![Bild_2023-01-25_172532130](https://user-images.githubusercontent.com/65771707/214619056-e6f6c283-28a4-40ec-b575-0c2e35d60d34.png)

**Comment by zax71** [comment]
Could you send your whole build.gradle file

**Comment by Dominic-Kaemereit** [comment, marked as answer]
build.gradle.kts
```
plugins {
    id ("com.github.johnrengelman.shadow") version "7.1.2"
}

repositories {
    maven {
        url = uri("https://jitpack.io")
    }
}

dependencies {
    implementation (project(":api"))
    implementation (project(":lib"))

    implementation ("com.github.Minestom:Minestom:-SNAPSHOT")
    implementation("net.kyori:adventure-api:4.12.0")
    implementation("net.kyori:adventure-text-minimessage:4.12.0")
    implementation ("org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.21")
}

tasks {
    build {
        dependsOn(shadowJar)
    }
}

tasks.withType(Jar::class) {
    manifest {
        attributes["Manifest-Version"] = "1.0"
        attributes["Main-Class"] = "core.server.main.MainKt"
    }
}
```

**Reply by zax71** [reply]
[Here](https://github.com/Ender-Cube/Spleef/blob/master/build.gradle.kts) is a basic example of a Minestom `build.gradle.kts` file, all this one does extra is import a few more dependancies but those can easily be removed

**Comment by FluxCapacitor2** [comment]
From the screenshot it looks like you're using IntelliJ. Make sure you've clicked the "Reload gradle" button.
IntelliJ docs: https://www.jetbrains.com/help/idea/work-with-gradle-projects.html#gradle_refresh_project
In the Gradle tool window, it's the top left button:
![image](https://user-images.githubusercontent.com/31071265/214997126-3d9b6688-3eaa-453f-a921-9f4d279dec5e.png)

If this does not solve your issue, I recommend replacing `-SNAPSHOT` with the first 10 characters of a commit hash in this repository. Using `-SNAPSHOT` with JitPack is known to be unreliable.

If it still doesn't work, let me know if you can run `gradlew build` in your terminal or through IntelliJ's "run anything" feature, and send the output here.
